stellargraph/core/graph.py

```diff
@@ -734,7 +735,7 @@ def node_features(self, nodes, node_type=None):
             node_type (hashable): the type of the nodes.
 
         Returns:
-            Numpy array containing the node features for the requested nodes.
+            tensorflow Tensor containing the node features for the requested nodes.
```

Comment:
I think we should consider carefully how we execute this transition; because this may break downstream users catastrophically. For instance, we could leave the default behaviour as returning numpy arrays but have a `tensor=True` flag. Do you have thoughts?

Reply:
@kjun9 I think this is the same point you raised. Yeah I think you're both right that we need to leave to a default option to return numpy arrays, I'm slightly more inclined to go with Kevin's idea of two seperate functions - so we know what type each function returns.